### PR TITLE
Create mobile dropdown menu

### PR DESCRIPTION
**Pull request by Lornakaboro**
In this activity we did the following :

- Created a new branch 
- Created the dropdown mobile-menu section in the index.html file
- Added the appropriate styling using style.css
- Created an script.js file for the JavaScript code
- Added a function ,so that when the hamburger menu is clicked the dropdown menu appears, and when the X-icon is clicked, the mobile menu disappears
- Added a blur-property so that when the mobile-menu is active the intro-section is blurred

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  
+  function openDropdownItem(menuElement, menuLink) {
+    document
+      .getElementById('nav-dropdown-menu')
+      .classList.toggle('hidden-dropdown-menu');
+    document
+      .getElementById('intro-section')
+      .classList.toggle('blur-intro-section');
+    menuElement.href = menuLink;
+  }
```
Comment (Qoosim) [inline script.js:18]: - [ ] Below are what you're expected to implement on this milestone and I gave pass mark those you implemented and red cross mark to those yet to be implemented. 
   -  When the user clicks the hamburger button, the mobile menu appears. :heavy_check_mark: 
   - When the user clicks the close (X) button, the mobile menu disappears. :heavy_check_mark: 
   - When the user clicks any of the mobile menu options, the mobile menu disappears. :heavy_check_mark: 
   - When the user clicks any of the mobile menu options, a correct part of the page is displayed. :negative_squared_cross_mark: 

-   [ ] Kindly work on the point marked red-cross. Please, take a look at how your project behave at this moment. :point_down: 


https://user-images.githubusercontent.com/26597432/201924536-f21a6910-7c3a-422a-bc25-8415648dc21e.mp4

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -9,6 +9,54 @@ body {
   background-color: #e5e5e5;
 }
 
+/* For drop down menu */
+.dropdown-menu {
+  position: absolute;
+  z-index: 10;
+  color: #fff;
```
Comment (Ginohmk) [inline style.css:16]: - [OPtional] Good job implementing changes required by the previous code reviewer, Your mobile menu goes to the appropriate page section :+1: However, as a tip, you can make it scroll smoothly to the particular section by adding a `smooth-scroll` in your CSS. [Link](https://www.w3schools.com/howto/howto_css_smooth_scroll.asp#section2)
```css
html {
  scroll-behavior: smooth;
}
```

- [OPtional] (Recommended) Please for a project that has javascript implementation, always add your javascript linter check. it helps you make use of javascript coding standards. [setup Eslint](https://github.com/microverseinc/linters-config/tree/master/javascript)

**Review by Ginohmk — APPROVED** [review]
Hi @Team, 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**